presentations.html

```diff
@@ -194,7 +194,7 @@ <h2 class="major" style="text-transform: none;">Lunch & Networking</h2>
 					<p>Catered Lunch to be served for all that register! <br>
 						&nbsp;&nbsp; <a target="_blank" href="https://www.eventbrite.com/e/decentralized-networks-summit-tickets-86959928487?aff=Cryptorado" target="_blank"><strong>Register HERE</strong></a>
 					</p>
-					<p> &nbsp;&nbsp;&nbsp;&nbsp;<em>Already an active user of web3 tech?<br>&nbsp;&nbsp;&nbsp;&nbsp;<strong>Please join us for a Think tank session</strong> 1:00pm - 2:00pm in the <a href="#workshops">workshop room</a></em> </p>
+					<p> &nbsp;&nbsp;&nbsp;&nbsp;<em>Already an active user of web3 tech?<br>&nbsp;&nbsp;&nbsp;&nbsp;<strong>Please join us for a Think tank session</strong> 1:00pm - 2:00pm in the <a href="workshops.html">workshop room</a></em> </p>
```

Comment:
Changed all these to the actual workshop.html file rather than #workshops as that wasn't working for me when developing locally... assuming this is correct?

Reply:
workshops is correct